cfme/tests/ansible/test_embedded_ansible_services.py

```diff
@@ -239,12 +239,22 @@ def test_service_ansible_playbook_bundle(ansible_catalog_item):
     BZ(1515841, forced_streams=['5.9'])
 ])
 def test_service_ansible_playbook_provision_in_requests(appliance, ansible_catalog_item,
-                                                        service_catalog):
+                                                        service_catalog, request):
     """Tests if ansible playbook service provisioning is shown in service requests."""
     service_catalog.order()
     cat_item_name = ansible_catalog_item.name
     request_descr = "Provisioning Service [{0}] from [{0}]".format(cat_item_name)
     service_request = appliance.collections.requests.instantiate(description=request_descr)
+
+    @request.addfinalizer
+    def _finalize():
+        _service = MyService(appliance, cat_item_name)
+        if service_request.exists():
+            service_request.wait_for_request()
+            service_request.remove_request()
+        if _service.exists:
```

Comment:
Required: exists(), as it is a function for MyService

Reply:
It should be a property, not a method. I pushed an appropriate PR https://github.com/ManageIQ/integration_tests/pull/6800.

Reply:
@quarckster, please change this back to property. #6800 has already been merged